d3log/src/tcp_network.rs

```diff
@@ -164,10 +176,11 @@ impl Transport for ArcTcpNetwork {
             {
                 Ok(_) => Ok(()),
                 // these need to get directed to a retry machine and an async reporting relation
-                Err(x) => panic!("send error {}", x),
+                Err(x) => {
+                    panic!("send error {}", x);
+                }
             }
         });
         (*self.n.lock().expect("lock").sends.lock().expect("lock")).push(completion);
```

Comment:
no. this is unfinished (in the current code as well). rather than have the dual error return, one which is synchronous on the call, and one which is asynchronous and actually reflects errors down the line, I thought it would be best to just make everything asynchronous. so these errors should get stuck in a channel (I guess), and a dedicated reader will take them out and .. I guess put them in a relation?